### ISSUE TITLE
install options - not google play store

### ISSUE THREAD

**Issue by Morse6690** [olvid-io/olvid-android#23]
Still loving Olvid. You all do great work.

Especially for degoogled devices it would be awesome to have the ability to make Olvid available to install via alternative app stores. For example Accrescent or f-droid. (https://github.com/accrescent/Accrescent) 
Users of these devices often have difficulty connecting to the Google play store.

Of course there is a need to still be able to pay for add ons to make the app profitable. 
Would it be possible to  do something like the Threema model. 
Firstly allow users to purchase licenses via your own website. These licenses could be used to access the paid calling feature etc.
Secondly make the app available on these alternative app stores, having the option to insert the license code to access extra features. 
This way free users can still access the app on all devices, with or without the play store, and they have the ability to support olvid financially.

**Comment by finiasz** [comment]
Hello @Morse6690, thanks for your message.

Publishing the app on F-Droid is indeed something we want to do, but unfortunately the F-Droid model forces us to recompile some libs directly from the sources instead of relying on packages pre-compiled by us. This is perfectly understandable, but for now we haven't had time to automate this process. The problematic libs are sqlite-jdbc and webrtc. For both of them we apply small patches to the source to support some features that are not supported by the distributed packages.

Regarding the purchasing of licences, it is indeed possible to purchase licences directly from our company, but we only sell them in "bulk", starting at a few dozen licences. These licences can be activated on degoogled devices. Selling licences directly from our website is also something we would love to do but haven't had time to focus on for now.

Right now, our main focus remains the release of new features!

**Comment by Morse6690** [comment]
Does Accrescent have similar requirements to f-droid?

**Comment by finiasz** [comment]
I am not familiar with accrescent, but it seems they even accept closed source applications, so there should not be any issue. However, their building process seems to be "slightly different" as pointed out in their doc https://accrescent.app/docs/guide/publish/building.html, so there might be some work to do on our side. We'll have a look at this!

**Comment by etam** [comment]
duplicate of #2?

**Comment by stemy2** [comment]
https://f-droid.org/packages/io.olvid.messenger.nogoogle

**Comment by Quantum-Future** [comment]
> Regarding the purchasing of licences, it is indeed possible to purchase licences directly from our company, but we only sell them in "bulk", starting at a few dozen licences. These licences can be activated on degoogled devices. Selling licences directly from our website is also something we would love to do but haven't had time to focus on for now.
> 
Now Olvid is available on F-Droid, great !
What about the possibility of buying individual licences from Olvid's website?

**Comment by finiasz** [comment]
We do have a store for this (https://store.olvid.io), but this is intended for businesses buying multiple licences and requiring a management console, so the price is higher than the individual in-app purchase...

**Comment by Quantum-Future** [comment]
Yes, I know this, but are you planning to let individual users buy a single license?
A user of an F-Droid version has no way to buy a license now.

**Comment by finiasz** [comment]
Yes, this is the plan. We would like to offer a pricing similar to purchases within Google Play or the iOS App Store for individuals. We simply need to implement this in our store. Hopefully we'll have time to do this soon.
